src/Umbraco.Web/PropertyEditors/GridPropertyEditor.cs

```diff
@@ -83,6 +83,9 @@ public override object FromEditor(ContentPropertyData editorValue, object curren
 
                 // editorValue.Value is a JSON string of the grid
                 var rawJson = editorValue.Value.ToString();
+                if (rawJson.IsNullOrWhiteSpace())
+                    return null;
+
```

Comment:
@Shazwazza does it also need null check on `Value`?

```
var rawJson = editorValue.Value?.ToString();
```

Reply:
We already check in the first line if editorValue.Value == null

Reply:
Ahh, didn't check the full source code.

Shouldn't this check be moved above these three lines?
https://github.com/umbraco/Umbraco-CMS/blob/v8/dev/src/Umbraco.Web/PropertyEditors/GridPropertyEditor.cs#L81-L83

Reply:
it could be moved yes, not a huge deal but if you want to submit a PR should be easy to merge.